runtime/lua/vim/shared.lua

```diff
@@ -20,6 +20,11 @@ vim.deepcopy = (function()
   local deepcopy_funcs = {
     table = function(orig)
       local copy = {}
+
+      if getmetatable(orig) == vim._empty_dict_mt then
```

Comment:
`if vim and getmetatable(orig) == vim._empty_dict_mt then`

Reply:
Sorry, actually `if vim._empty_dict_mt ~= nil and getmetatable(orig) == vim._empty_dict_mt then`